### PR TITLE
Sidebar based component changes

### PR DESCRIPTION
**Pull request by rakeshyc**
Css changes based on sidebar changes

### CODE REVIEW

File: scss/product/components/_badge.scss [hunk 1]
```diff
@@ -90,4 +94,15 @@
   display: inline-flex;
   align-items: center;
   justify-content: center;
-}
\ No newline at end of file
+}
+
+// notification with a dot
+ .badge-notification-dot {
```
Comment (Pallavi-PH) [inline scss/product/components/_badge.scss:100]: Can you share the screenshot of badge-notification-dot?

File: scss/product/components/_badge.scss [hunk 2]
```diff
@@ -59,6 +59,10 @@
   height: 12px;
   width: 12px;
   vertical-align: middle;
+  &.circle-small{
```
Comment (Pallavi-PH) [inline scss/product/components/_badge.scss:62]: And circle-small?

**Comment by rakeshyc** [comment]
@Pallavi-PH 
image

File: scss/product/components/_badge.scss (hunk 1 above)
Comment (rakeshyc) [reply scss/product/components/_badge.scss:100]: i think we can move this style to sidebar.scss right? it wont make any sense adding it in badge.scss @Pallavi-PH

File: scss/product/components/_badge.scss (hunk 2 above)
Comment (rakeshyc) [reply scss/product/components/_badge.scss:62]: badge-sm class can be used here, i will remove this style

**Comment by Pallavi-PH** [comment]
> @Pallavi-PH
> image

This can be added as `badge-icon-notification` Because this can be used with other icons too if need. You can add this in badge.scss itself. Also, add it in badge.mdx file for display

File: docs/products/badge.mdx [hunk 3]
```diff
@@ -26,6 +26,15 @@ import { Playground, PropsTable } from 'docz'
 <span className="badge badge-notification mr-2">53</span>
 </Playground>
 
+
+### Badge notification with icon
+<Playground>
+  <span className="badge badge-notification-dot">
```
Comment (Pallavi-PH) [inline docs/products/badge.mdx:32]: can we call this `badge-icon-notification` instead?

**Review by Pallavi-PH — APPROVED** [review]
LGTM

**Review by avishnu — APPROVED** [review]
Thanks.

**Pull request merged**